src/Build/BuildCheck/Infrastructure/NullBuildCheckManager.cs

```diff
@@ -47,4 +45,7 @@ public void YieldProject(BuildCheckDataSource buildCheckDataSource, BuildEventCo
 
     public void ResumeProject(BuildCheckDataSource buildCheckDataSource, BuildEventContext buildEventContext)
     { }
+
+    public Dictionary<string, TimeSpan> CreateAnalyzerTracingStats() => throw new NotImplementedException();
```

Comment:
Is there a chance not to throw the exception here? 
Just to be on the safe side in case it is invoked, I do not think we should raise the exception. 
@JanKrivanek what do you think?

Reply:
I changed to return an empty list, so it does not throw and not change to rest of the code to account for a null value.